src/c++/perf_analyzer/main.cc

```diff
@@ -1189,14 +1243,38 @@ main(int argc, char** argv)
   if (!max_threads_specified && target_concurrency) {
     max_threads = 16;
   }
+  if (kind == cb::BackendKind::TRITON_LOCAL) {
+    std::cout << " USING C API: only default functionalities supported "
+              << std::endl;
+    if (!target_concurrency) {
+      std::cerr << "Target concurrency not yet supported by C API" << std::endl;
```

Comment:
This message seems wrong. Should be that only target concurrency is supported... whatever the opposite of that is is not supported.